bin/update_currencies.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+use Buzz\Client\ClientInterface;
+use Buzz\Message\Request;
+use Buzz\Message\Response;
+
+require_once __DIR__.'/../vendor/autoload.php';
+
+final class CurrencyUpdater
```

Comment:
Not really (and I didn't put much thought into it). Just there to separate things out a bit.